### DISCUSSION TITLE
An option to copy values to clipboard

### DISCUSSION THREAD

**Discussion by jessicah** [datalust/seq-tickets#2144] [Ideas]
Sometimes, when the value is quite large, there's a lightbulb icon that shows up, allowing to copy the value (Copy value & Copy raw JSON). Can this be added for all values?

In particular, I'm logging raw JSON when I hit a parsing error, but it's quite large, and there's no auto-scrolling whilst trying to select it, so it's very difficult to copy. Atm, I resort to browser devtools and copying it out that way.

**Comment by nblumhardt** [comment]
Thanks for the suggestion, makes sense. We'll take a look 👍

**Reply by nblumhardt** [reply]
Thanks again for raising this, we're not planning to implement this in the current release, but we'll keep it on the radar.
